tests/documents/test_document_models.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+from django.contrib.contenttypes.models import ContentType
+from django.core.exceptions import ValidationError
+
+from adhocracy4.comments import models as comments_models
+from apps.documents.models import Document
+
+
+@pytest.mark.django_db
+def test_paragraph_save(paragraph):
+    assert '<script>' not in paragraph.text
```

Comment:
what does this test do?
i would make this more explicit by creating a paragraph instance with a script tag in the context of this test instead of using the more decoupled value set in the default paragraph factory